content/index.ja.md

```diff
@@ -0,0 +1,700 @@
+-   [ガイドライン 1.8 - ビューポート内のオリエンテーション](https://www.w3.org/TR/UAAG20/#gl-viewport-orient)
+-   [ガイドライン 1.9 - 代替ビュー](https://www.w3.org/TR/UAAG20/#gl-alternative-views)
+-   [ガイドライン 1.10 - 要素情報](https://www.w3.org/TR/UAAG20/#gl-info-link)
+
+**ATAG**
+
+-   [原則 A.1: オーサリングツールのユーザーインターフェイスは適用可能なアクセシビリティガイドラインに従う](https://www.w3.org/TR/ATAG20/#principle_a1)
+-   [ガイドライン A.3.7: (オーサリングツールのユーザーインターフェイス用) プレビューが市場に出回っているユーザーエージェントと同等以上にアクセシブルであることを保証する](https://www.w3.org/TR/ATAG20/#gl_a37)
+-   [パート B. アクセシブルなコンテンツの制作をサポートする](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### 区別可能性に関連する記事
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Lee、色覚異常を持つオンラインショッパー](/people-use-web/user-stories/#shopper)
+-   [Martine、聴覚障害を持つオンライン学生](/people-use-web/user-stories/#onlinestudent)
+-   [Ilya、視覚障害を持つ上級職員](/people-use-web/user-stories/#accountant)
+-   [Yun、低視力、手の震え、および軽度の短期記憶障害を持つ退職者](/people-use-web/user-stories/#retiree)
+-   [Kaseem、聴覚障害と視覚障害を持つ10代の若者](/people-use-web/user-stories/#teenager)
+
+{% include excol.html type="end" %}
+
+## 操作可能なユーザー・インターフェースとナビゲーション {#operable}
+
+### キーボードからの操作が可能 {#keyboard}
+
+多くの人々はマウスを使用せず、キーボードに頼ってウェブとやり取りします。これには、フォームコントロール、入力欄、その他のユーザーインターフェイスコンポーネントを含む、すべての機能へのキーボードアクセスが必要です。
+
+キーボードのアクセシビリティ:
+
+-   マウスで利用可能なすべての機能がキーボードでも利用可能である
+-   キーボードフォーカスがコンテンツの任意の部分に閉じ込められない
+-   ウェブブラウザ、オーサリングツール、およびその他のツールがキーボードサポートを提供する
+
+この要件を満たすことで、人間工学的なレイアウトのキーボード、画面上のキーボード、スイッチデバイスなどの代替キーボードを使用する人々を含む、キーボードユーザーが支援されます。また、音声認識（音声入力）を使用してウェブサイトを操作したり、キーボードインターフェースを通じてテキストを入力したりする人々にも役立ちます。
+
+{% include excol.html type="start" %}
+
+#### キーボード・アクセシビリティに関するアクセシビリティ要件（技術仕様へのリンク）
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+**WCAG**
+
+-   [ガイドライン 2.1 - キーボードでアクセス可能](https://www.w3.org/WAI/WCAG22/quickref/#keyboard-accessible)
+
+**UAAG**
+
+-   [ガイドライン 2.1 - キーボードアクセス](https://www.w3.org/TR/UAAG20/#gl-keyboard-access)
+-   [ガイドライン 2.2 - 順序付けられたナビゲーション](https://www.w3.org/TR/UAAG20/#gl-sequential-navigation)
+-   [ガイドライン 2.3 - 直接ナビゲーションとアクティベーション](https://www.w3.org/TR/UAAG20/#gl-direct-navigation-and-activation)
+-   [ガイドライン 2.11 - その他の入力デバイス](https://www.w3.org/TR/UAAG20/#gl-other-devices)
+
+**ATAG**
+
+-   [原則 A.1: オーサリングツールのユーザーインターフェイスは適用可能なアクセシビリティガイドラインに従う](https://www.w3.org/TR/ATAG20/#principle_a1)
+-   [ガイドライン A.3.1: (オーサリングツールのユーザーインターフェイス用) オーサリング機能へのキーボードアクセスを提供する](https://www.w3.org/TR/ATAG20/#gl_a31)
+-   [パート B. アクセシブルなコンテンツの制作をサポートする](https://www.w3.org/TR/ATAG20/#part_b)
+
+{% include excol.html type="end" %}{% include excol.html type="start" %}
+
+#### キーボード・アクセシビリティに関連する記事
+{:.no_toc}
+
+{% include excol.html type="middle" %}
+
+-   [Alex, 再発性ストレス障害を持つ記者](/people-use-web/user-stories/#reporter)
+-   [Ilya, 盲目のシニアスタッフ](/people-use-web/user-stories/#accountant)
+
+{% include excol.html type="end" %}
+
+### コンテンツを読み、利用するのに十分な時間が必要なユーザー {#time}
+
+一部の人々は、テキストを入力したり、指示を理解したり、コントロールを操作したり、ウェブサイトでのタスクを完了するために他の人よりも時間がかかる場合があります。
```

Comment:
@takuan517 This part seems to have been reworded. Would you find a formulation closer to the original version?

Reply:
@remibetin 
fixed 
https://github.com/w3c/wai-accessibility-principles/pull/50/commits/1f48e9020f178cfc2a3b4cb20132df75873b44ca